.goreleaser-template.yaml

```diff
@@ -40,6 +40,16 @@ builds:
       - s390x
     goarm:
       - "6"
+    ignore:
```

Comment:
Is this key duplicate?

Reply:
No, this is in a separate array item. `builds` (the top-level key) is an array of two entries; one dict with `id=builds-linux` and one dict with `id=builds-other`. We have to cross-exclude the OSes as `goos` is templated now.